### ISSUE TITLE
Invalid role on Console session suspend status icons

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#12449]
### System details

    RStudio Edition : Server (or Server Pro)
    RStudio Version : 2022.12.0+353
    OS Version      : all
    R Version       : all

### Steps to reproduce the problem

The icons shown in the Console toolbar when a session is suspended (`#r_session_suspended_console`) or when a session cannot be suspended (e.g. if you run `sleep 5000` in the Terminal pane, `#r_session_suspend_blocked_console`), both have `role="alert"` on them. This is nonsensical (invalid HTML) and breaks screen reader support for these images.

![Screenshot of console pane with an arrow pointing to the session suspended icon](https://user-images.githubusercontent.com/10569626/208153674-fcbef7d8-fc69-432d-b618-b41e25f5e90e.png)
![Screenshot of console pane with an arrow pointing to the session cannot suspend icon](https://user-images.githubusercontent.com/10569626/208153676-764b90ca-1d93-4630-b891-a4dbae82767c.png)

### Describe the problem in detail

If you try to navigate to one of these icons using VoiceOver on macOS, you cannot. The role needs to be left at the default (`img`).

### Describe the behavior you expected

These images should be accessible to a screen reader user since they communicate status information.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
Verified in RStudio Server 2023.05.0-daily+124 on Ubuntu 20.04 via Chrome on MacOS 13.2.

Using Inspect element:

*Cherry Blossom*
![Image](https://user-images.githubusercontent.com/31009952/227579536-7dcdf3fb-76ac-4acf-bf7a-2841af6c941b.png)

*Mountain Hydrangea*
![Image](https://user-images.githubusercontent.com/31009952/227579610-cf16693d-e9d7-4613-be14-5b7f877ef80b.png)
